x-pack/legacy/server/lib/check_license/check_license.js

```diff
@@ -14,6 +14,10 @@ import {
 } from '../../../common/constants';
 
 export function checkLicense(pluginName, minimumLicenseRequired, xpackLicenseInfo) {
+  if (!minimumLicenseRequired) {
```

Comment:
As we don't have defaults, I added this check in order to make the error clearer to the consumer. We might remove it once we are all in Typescript world 😊